explorer/README.md

```diff
@@ -0,0 +1,105 @@
+# Explorer
+
+## Requirements
+
+- [Erlang 26](https://github.com/asdf-vm/asdf-erlang)
+- [Elixir 1.16.2](https://elixir-ko.github.io/install.html), compiled with OTP 26
+- [Phoenix 1.7.12](https://hexdocs.pm/phoenix/installation.html)
+
+## Local development
+
+### Set up environment variables
+
+Run the following command to set up the environment variables:
+
+```sh
+make create_env
+```
+
+This will create a `.env` file in the `/explorer` directory of the project. The `.env` file will contain the following variables:
+
+| Variable | Description |
+| -------- | ----------- |
+| `RPC_URL` | The RPC URL of the network you want to connect to. |
+| `ENVIRONMENT` | The environment you want to run the application in. It can be `devnet`, `holesky` or `mainnet`. |
+
+### Run devnet environment (optional)
+
+If you want to run the devnet environment, you can run the following command in another terminal:
+
+```sh
+cd ..
+make anvil-start
+```
+
+Then in another terminal, you can run the following command to run the operator:
+
+```sh
+make operator-full-registration
+make operator-start
+```
+
+Then, in another terminal, run the following command to start the aggregator:
+
+```sh
+make aggregator-start
+```
+
+Finally, to have a task running in the devnet, you can run the following command:
+
+```sh
+make send-plonk_bls12_381-proof-loop
+```
+
+### Run the frontend
+
+Set up your environment variables:
+
+```sh
+export RPC_URL=your_rpc_url
+```
```

Comment:
is this step necessary now that we have the `.env` file?

Reply:
Nope, its now included in the env